Add missing tasks from prompt source

yeganehkordi opened the pull request.

yeganehkordi committed:
add missing tasks

yeganehkordi committed:
Update task1290_xsum_summarization.json

yeganehkordi committed:
Update task1291_multi_news_summarization.json

Palipoor commented:
- In `task1287_glue_qqp_paraphrasing` and `task1288_glue_mrpc_paraphrasing` : Maybe add "and have the same answer" to the definition.
- In `task1291_multi_news_summarization`: What is the |||| separating from what? Also, can you convert ’ characters to ' s?
- In `task1393_superglue_copa_text_completion`: I think the definition is a bit vague. Can you reword it?

Other than these and the comments on the readme file, this looks good to me!

yizhongw commented:
Comments on the categories and sources:
* In `task1286_openbookqa_question_answering.json`, change the source from`openbookqa/main` to `openbookqa`.
* In `task1287_glue_qqp_paraphrasing.json`, change the source from`glue/qqp` to `qqp`, change the category from `Paraphrasing` to `Text Matching` (because I use paraphrasing mainly for the paraphrase generation).
* In `task1288_glue_mrpc_paraphrasing.json`, change the source from `glue/mrpc` to `mrpc`, remove `https://gluebenchmark.com/` from the URL, change the category from `Paraphrasing` to `Text Matching`.
* In `task1289_trec_classification.json`, change the category from `Text Categorization` to `Question Understanding`.
* In `task1292_yelp_review_full_text_categorization.json`, change the category from `Text Categorization` to `Sentiment Analysis`.
* In `task1392_superglue_multirc_answer_verification.json`, change the source from `MultiRC` to `multirc`.
* In `task1393_superglue_copa_text_completion.json`, change the category from `Text Completion` to `Cause Effect Classification`.

yeganehkordi committed:
update

yeganehkordi commented:
Thanks! I've addressed your comments.
@Palipoor  I can't see your comments on the readme file (I haven't entered the categories, and other data intentionally there, we can update them with script later). 
In `task1291_multi_news_summarization` each article consists of many pieces of news which are separated with ||||.
